### PR TITLE
Resolves CoarchyM-100290: Fix paste into empty story

### PR DESCRIPTION
**Pull request by marwand**
Changes:

- Refactored ActivityPaste form into a new file to avoid code redundancy
- Updated paste#Activities to support pasting into an empty story
- Added a generic Localized error message

### CODE REVIEW

File: screen/coarchy/cointernal/Process/EditProcessStory/PasteActivity.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+                                    <field name="activityText">
+                                        <default-field title="">
+                                            <display/>
+                                        </default-field>
+                                    </field>
+
+                                    <field name="insertFirst">
+                                        <default-field title="">
+                                            <link url="pasteActivities" text="Insert" btn-type="success" parameter-map="
```
Comment (acetousk) [inline screen/coarchy/cointernal/Process/EditProcessStory/PasteActivity.xml]: The insert button text should include the number of activities like the paste widget in the actions menu

File: screen/coarchy/cointernal/Process/EditProcessStory.xml [hunk 2]
```diff
@@ -391,7 +395,12 @@ along with this software (see the LICENSE.md file). If not, see
                             icon="fa fa-trash" btn-type="danger" text="Delete" tooltip="Delete Activity"
                             parameter-map="[processStoryActivityId:processStoryActivityId,activityId:activityId,actorIdList:actorIdList,pageIndex:pageIndex]"/>
                 </button-menu></default-field>
-                <last-row-field><link url="newParagraph" text="⏎ Paragraph" tooltip="Add Paragraph Break"
+                <last-row-field>
+                    <dynamic-dialog id="PasteActivity" button-text="Paste" transition="PasteActivity"
```
Comment (acetousk) [inline screen/coarchy/cointernal/Process/EditProcessStory.xml]: If there's multiple buttons, let's put the buttons in a button-menu widget

File: screen/coarchy/cointernal/Process/EditProcessStory/PasteActivity.xml (hunk 1 above)
Comment (marwand) [reply screen/coarchy/cointernal/Process/EditProcessStory/PasteActivity.xml]: Do you have a specific label in mind? I used Paste ${activityCount}

Comment (acetousk) [reply screen/coarchy/cointernal/Process/EditProcessStory/PasteActivity.xml]: That's fine

**Pull request merged**